tests/integration/cattletest/core/test_secrets.py

```diff
@@ -204,10 +213,16 @@ def test_service_secrets_fields(context, secret_context):
         {
             "name": "my_secret2",
             "secretId": secret2.id,
-            "mode": "444",
+            "mode": "0444",
```

Comment:
@StrongMonkey Shall we pass 444 and validate it is 0444 in the exported yml? To actually exercise the code in export that appends 0

Reply:
@alena1108 sure, changed